components/automate-ui/src/app/page-components/profile/profile.component.html

```diff
@@ -16,16 +16,14 @@
       <li class="dropdown-list-item">
         Signed in as <strong class="dropdown-display-name">{{ displayName }}</strong>
       </li>
-      <app-authorized [allOf]="['/auth/users/{username}', 'get', userName]">
```

Comment:
Was the permissioning not appropriate here? Wondering why it was removed.

Reply:
I was under the impression that this was only here to check if the user was a local user or not. This was to decide if we should show the profile link. We now have the `isLocalUser` variable that does this check.

Reply:
No, it is not checking local-or-not status.  That was a standard permission check to determine if the current user (local or not) have permission to do a get on the specified user (which must be a local user).

Reply:
What is the reason for the check? It looked like to me to see if the profiles page could be displayed.